docs/CHANGELOG.asciidoc

```diff
@@ -73,6 +74,46 @@ This section summarizes the changes in each release.
 //=== Known Issues
 ////
 
+
+[[release-notes-6.3.2]]
+== {kib} 6.3.2
+coming[6.3.2]
+
+[float]
+=== Bug Fixes
+Discover::
+* [Fixes #19445] Removes outdated Aria attribute {pull}20532[#20532]
+
+Monitoring::
+* Fixes issues with sorting and filtering of {es} nodes {pull}20383[#20383]
++
+This issue fixes the sorting and filtering of rows in the table of {es} nodes 
+in the {kib} Monitoring UI.
+
+Platform::
+* Defaults to running the {es} functional test server from a snapshot locally 
+in the {xpack} directory {pull}20585[#20585]
+
+Sharing::
+* [Fixes #20635] Changes the scale factor when upgrading panels with 
+margins {pull}20684[#20684]
++
+This change reduces the scale factor to 4 when migrating panels that are in 
+a dashboard with margins.
+
+* Fixes issue with running Chromium headless over HTTP when accessing {kib} 
+publicly via HTTPS {pull}20528[#20528]
```

Comment:
I think this PR (https://github.com/elastic/kibana/pull/20528) is specific to Reporting, so it might be helpful to put it under a "Reporting:" section, unless @chrisdavies prefers otherwise.